src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,441 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Transforms;
+using TensorFlow;
+
+[assembly: LoadableClass(TensorflowTransform.Summary, typeof(TensorflowTransform), typeof(TensorflowTransform.Arguments), typeof(TensorflowTransform),
+    TensorflowTransform.UserName, "CopyColumns", "CopyColumnsTransform", TensorflowTransform.ShortName, DocName = "transform/CopyColumnsTransform.md")]
+
+[assembly: LoadableClass(TensorflowTransform.Summary, typeof(TensorflowTransform), null, typeof(SignatureLoadDataTransform),
+    TensorflowTransform.UserName, TensorflowTransform.LoaderSignature)]
+
+namespace Microsoft.ML.Transforms
+{
+    public sealed class TensorflowTransform : RowToRowMapperTransformBase, IDisposable
```

Comment:
>IDisposable [](start = 75, length = 11)

That is incorrect. Our transforms should never be disposable.

Reply:
Took a look at DnnImageFeaturizerTransform and it just lets GC finalize disposable CNTK.Function. Also checked CntkPredictorBase and it does the same for its public Function Model property.

---
In reply to: [209664938](https://github.com/zeahmed/machinelearning/pull/3#discussion_r209664938) [](ancestors = 209664938)